src/index.html

```diff
@@ -19,9 +19,381 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header__nav-item">
```